pkgs/servers/sql/materialize/default.nix

```diff
@@ -68,8 +68,12 @@ rustPlatform.buildRustPackage rec {
     "--skip test_client"
     "--skip test_client_errors"
     "--skip test_client_all_subjects"
+    "--skip test_client_subject_and_references"
     "--skip test_no_block"
     "--skip test_safe_mode"
+    # this test is broken on 0.15.0
+    # TODO: re-add it in a subsequent release
```

Comment:
Upstream is working on a fix?
Do they have an issue to track it?

Reply:
disclaimer: I'm working for materialize.

This is already fixed in a pending PR. Despite the test's name this doesn't actually test anything to do with threading, it tests whether the executable can parse the command line parameter for specifying the number of threads correctly, and passing a non-UTF8 value produces a different error than what the test expects.

tl;dr: this test can be safely ignored without user impact and will be removed in the next release

Reply:
Great, thanks! Sounds good.

I trust that you tested it adequately then.